db/revision.go

```diff
@@ -101,6 +101,8 @@ func (body Body) getExpiry() (uint32, error) {
 	return 0, nil
 }
 
+const nonJSONPrefix = byte(1)
```

Comment:
Can you add a comment that explains this out a little more?  Or ideally references some docs, if there is any.  As it stands, it feels a bit cryptic.